odoo/addons/base/models/res_users.py

```diff
@@ -1091,6 +1091,26 @@ def write(self, values):
             self._check_one_user_type()
         return res
 
+    def _apply_group(self, implied_group):
+        """
+        Helper function to an an inherited/implied group to a group (sudo is required)
+
+        @param implied_group: The implied group to add
+        """
```

Comment:
```suggestion
        """Add the given group to the groups implied by the current group(s)

        :param implied_group: The implied group to add
        """
```
`@param` notation is for javascript, in python the correct basic notation is `:param type name: explanation`.
The sudo comment is IMHO not useful (can be seen on usages and during testing).
Also, ideally, you should follow Python docstring conventions (can be done with pydocstyle, but it gives red warnings for most of odoo  code :D).